### PR TITLE
TestData: support dropping points from random walk data and send interval

### PR DESCRIPTION
**Pull request by ryantxu**
Given the effort and solutions we are exploring for inserting null values into Prometheus results, it would be nice to have testdata that does the same thing.  This PR adds a "Drop (%)" field to testdata that will randomly drop values.  This will set the interval field config all the time:

image

image

Once the new support is added, we expect the dropped values to become breaks in the graph automatically

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -47,14 +47,14 @@ require (
...
-	github.com/google/go-cmp v0.5.6
+	github.com/google/go-cmp v0.5.7
 	github.com/google/uuid v1.3.0
 	github.com/google/wire v0.5.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/gosimple/slug v1.9.0
 	github.com/grafana/cuetsy v0.0.0-20211119211437-8c25464cc9bf
 	github.com/grafana/grafana-aws-sdk v0.9.1
-	github.com/grafana/grafana-plugin-sdk-go v0.123.0
+	github.com/grafana/grafana-plugin-sdk-go v0.124.1-0.20220127163549-4ffddbc40722
```
Comment (ryantxu) [inline go.mod:57]: @toddtreece do we plan to release the SDK bump soon?

Comment (toddtreece) [reply go.mod:57]: yeah, i will do that tonight. i was waiting to see if we needed the 8.3.6 backport

Comment (toddtreece) [reply go.mod:57]: https://github.com/grafana/grafana/pull/44685

**Review by dprokop — APPROVED** [review]
Works as described :)

**Pull request merged**